python/xorbits/pandas/pandas_adapters/tests/test_pandas_adapters.py

```diff
@@ -113,30 +113,6 @@ def test_pandas_dataframe_methods(setup):
             "zoo": ["x", "y", "z", "q", "w", "t"],
         }
     )
-    df = xpd.DataFrame(raw)
```

Comment:
Please replce the removed test cases with another fallback method.

Reply:
The point for this test case is to see if xorbits prints a warning correctly when some fallback method being called.

Since you have implemented `pivot`, you can choose another method, which has not been implemented yet, for testing.

Reply:
I'd prefer fix this problem in this PR. Otherwise, once this PR got merged, we wouldn't know if the fallbacks are printing warnings correctly in some following PRs. What do you think?

Reply:
> I'd prefer fixing this problem in this PR. Otherwise, once this PR got merged, we wouldn't know if the fallbacks are printing warnings correctly in some following PRs. What do you think?

+1. Just select a function that will fallback to test if it works, such as `df.asof`.